earth_enterprise/src/fusion/tools/gee_test/publish_historical.sh

```diff
@@ -23,15 +23,20 @@ echo "Using asset root: $ASSET_ROOT"
 
 /opt/google/bin/geserveradmin --adddb "$ASSET_ROOT/Tutorial/Databases/SFinset_1.kdatabase/gedb.kda/ver001/gedb"
```

Comment:
This works just fine, but may be use a simpler path for the database:
/opt/google/bin/geserveradmin --adddb Tutorial/Databases/SFinset_1.kdatabase

So we don't have to point to the low level file ... If you need that specific version, i would use:
/opt/google/bin/geserveradmin --adddb Tutorial/Databases/SFinset_1.kdatabase?version=1